canonical-webteam/global-nav
# Jump when opening on small screens

## Done
Jump to the bottom of the list when opening on small screens. Also make the button white on small screens.

## QA
- Pull branch 
- Run `gulp build`
- Open index.html
- Go to small screen and scroll to the bottom
- Click Canonical websites button
- See that the menu opens within the view without you needing to scroll again
